package.json

```diff
@@ -21,12 +21,12 @@
     "parameter store",
     "env"
   ],
-  "author": "Jared Short <jshort@trek10.com> (https://www.trek10.com)",
+  "author": "Sneha Nagole <sneha.nagole@tabordasolutions.com>",
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/trek10inc/serverless-secrets/issues"
   },
-  "homepage": "https://github.com/trek10inc/serverless-secrets",
```

Comment:
I recommend reverting this for a cleaner PR going to the upstream repo.

Reply:
Changes made.